tests/FSharp.AWS.DynamoDB.Tests/MultipleKeyAttributeTests.fs

```diff
@@ -0,0 +1,78 @@
+module FSharp.AWS.DynamoDB.Tests.MultipleKeyAttributeTests
+
+open System
+
+open Swensen.Unquote
+open Xunit
+
+open FSharp.AWS.DynamoDB
+open FSharp.AWS.DynamoDB.Scripting
+
+[<AutoOpen>]
+module MultiKeyTypes =
+
+    type InverseKeyRecord =
+        { [<HashKey>]
+          [<GlobalSecondaryRangeKey(indexName = "Inverted")>]
+          PrimaryKey: string
+          [<RangeKey>]
+          [<GlobalSecondaryHashKey(indexName = "Inverted")>]
+          SortKey: string }
+
+    type SharedRangeKeyRecord =
+        { [<HashKey>]
+          HashKey: string
+          [<GlobalSecondaryHashKey(indexName = "GSI1")>]
+          GSI1 : string
+          [<GlobalSecondaryHashKey(indexName = "GSI2")>]
+          GSI2 : string
+          [<GlobalSecondaryRangeKey(indexName = "GSI1")>]
+          [<GlobalSecondaryRangeKey(indexName = "GSI2")>]
+          SortKey: string }
+
+type ``Inverse GSI Table Operation Tests`` (fixture : TableFixture) =
+
+    let rand = let r = Random() in fun () -> int64 <| r.Next()
```

Comment:
.NET 7 has a Shard.Random which is threadsafe (and also means the seed can't be identical if lots of instances of the TestClass get constructed in the same timeslice)